### PR TITLE
Small manual adjusts for docs and bundle for v0.29.0

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -21,11 +21,11 @@ metadata:
     containerImage: quay.io/opentelemetry/opentelemetry-operator
     createdAt: "2020-12-16T13:37:00+00:00"
     description: Provides the OpenTelemetry components, including the Collector
-    operators.operatorframework.io/builder: operator-sdk-v1.9.0
+    operators.operatorframework.io/builder: operator-sdk-v1.8.0+git
```
Comment (VineethReddy02) [inline bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:24]: Any specific reason for this change? On my PC `make bundle` doesn't generate this change. :/ 
I am using `operator-sdk` of version `v1.9.0`

Comment (jpkrohling) [reply bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:24]: Strange, this is what I get:

```console
$ operator-sdk version
operator-sdk version: "v1.9.0", commit: "205e0a0c2df0715d133fbe2741db382c9c75a341", kubernetes version: "1.20.2", go version: "go1.16.5", GOOS: "linux", GOARCH: "amd64"

$ make bundle
/home/jpkroehling/go/bin/controller-gen "crd:trivialVersions=true,preserveUnknownFields=false,generateEmbeddedObjectMeta=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
/usr/bin/operator-sdk generate kustomize manifests -q
WARN[0000] Skipping definitions parsing for API {}: Go type not found 
/home/jpkroehling/bin/kustomize build config/manifests | /usr/bin/operator-sdk generate bundle -q --overwrite --manifests --version "0.29.0-1-gc7909cd"  
/usr/bin/operator-sdk bundle validate ./bundle
INFO[0000] Found annotations file                        bundle-dir=bundle container-tool=docker
INFO[0000] Could not find optional dependencies file     bundle-dir=bundle container-tool=docker
INFO[0000] All validation tests have completed successfully 

$ grep operator-sdk  bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml 
    operators.operatorframework.io/builder: operator-sdk-v1.8.0+git
```

Comment (VineethReddy02) [reply bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:24]: I tried `make bundle` multiple times it was generating `operator-sdk-v1.9.0` with operator-sdk version `v1.9.0`

But after uninstalling `operator-sdk` and re-installing it. The issue got fixed now I see `operator-sdk-v1.8.0+git`. Very strange. Apologies for the confusion.

**Comment by VineethReddy02** [comment]
LGTM!

File: bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (owais) [reply bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:24]: I think this depends on how you install operator-sdk. For example, brew vs github releases. One uses 1.8.0+git while the other uses 1.9.0 (correctly). I don't remember which is which.

**Review by owais — APPROVED** [review]

File: bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (jpkrohling) [reply bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:24]: I downloaded the binary from GitHub and it uses what is in this PR.

Comment (VineethReddy02) [reply bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:24]: Yup, In the first run I upgraded `operator-sdk` using `brew` which added version as `v1.9.0`, In the second attempt I uninstalled using brew and re-installed it using Github downloads which added the version as `operator-sdk-v1.8.0+git`.

**Pull request merged**